### PR TITLE
Update Export scripting test

### PR DESCRIPTION
**Pull request by Karl-G1**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-routing-and-faulting-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

The Export unit test for Routing and Faulting was failing after the high-level scripting APIs were added. This change updates the exported data used to compare with the data returned by the scripting API.

### Why should this Pull Request be merged?

Update the scripting test to pass.

### What testing has been done?

Ran tests in dev environment with LabVIEW and VeriStand 2017

### CODE REVIEW

**Comment by rtzoeller** [comment]
This test is still failing on my machine, which makes me think we have different VI packages installed. This is concerning, because I'm questioning what version of these packages we have installed on the build machine.

Can you validate that you have the same versions of the packages as listed in this pull request (which is what I have installed)? https://github.com/ni/niveristand-routing-and-faulting-custom-device/pull/167

It does not look like that is what is currently being hosted in the list of packages used by the build machine...

**Comment by rtzoeller** [comment]
The test as it exists on master passes on my machine. I think the test asset was updated correctly, but the ATS and build machines (and your dev VM) are using outdated versions of the advanced sysdef resources.

**Comment by rtzoeller** [comment]
Thanks @buckd for doing some rapid testing to help figure out what was going on!

The TL;DR appears to be that this pull request can be closed, and that we don't have anything to worry about (although that's not the case at first glance, and we got lucky) other than updating the files we are installing on the build and test machines.

---

It looks like the 19.2 release was built against `ni_lib_advanced_sysdef_api-1.2.0.1.vip`, which was created as a workaround when the source for the Advanced SysDef repository had moved to LabVIEW 2019. This release included new support for preserving item order.

When the Advanced SysDef repository was moved to 2019, [some additional changes](https://github.com/NIVeriStandAdd-Ons/VeriStand-Addon-System-Definition-Utilities/commit/dafbf5f50c3208eee0fa03fced9c3c2ba0e56a3c) were made to the repo to handle relative paths. When the changes to preserve item order were merged after the 2019 source was saved-for-previous, this was exported as `ni_lib_advanced_sysdef_api-1.1.1.12.vip`, which is what the tests were created with.

In the 1.2.0.1 case (and 1.1.1.8), `TreeItem.Property.Get.vi` uses an empty variant as the value for an attribute to indicate that the string is actually a path. In 1.1.1.12, a bool is used as the value of the attribute, and the bools value indicated whether it is a relative path. Fortunately, the variant used by 1.2.0.1/1.1.1.8 should be handled correctly by 1.1.1.12.

This means that the 19.2 release was built against a slightly mystery-meat version of the Advanced SysDef API. Luckily, it appears to be internally consistent, and the version we intended to build with and should build with moving forward (1.1.1.12) is backwards compatible.

**We should define a better process for updating the package versions, so this doesn't happen again.** We should also do some sanity testing, to confirm what I wrote is actually correct.

**Comment by Karl-G1** [comment]
The build machines and my test VM are updated with the correct VI Package versions. The test now passes as it exists in master, so closing this PR.
